### ISSUE TITLE
Resizable flexboxes for playback, settings components

### ISSUE THREAD

**Issue by csm-kb** [Geoxor/Amethyst#165]
**Describe the idea you'd like for Amethyst to have**
Having poked at the code and given it an initial `dev` run, my absolute first thought was "Huh... the settings and playback drawers should be resizable." Think Unity editor in terms of its resizable drawers, or Discord voice channel textbox (you can resize it, surprisingly).

This is something we might be able to implement re. an initial jaunt to [this stackoverflow](https://stackoverflow.com/questions/46931103/making-a-dragbar-to-resize-divs-inside-css-grids), but in the Vue way, with default widths as well ala `w=64` for [SettingsBar](https://github.com/Geoxor/amethyst/blob/master/src/renderer/components/SettingsBar.vue#L13), and so on. I'm experimenting (between works) to see if I can't expand or adapt the [ResizableDiv](https://github.com/Geoxor/amethyst/blob/master/src/renderer/components/ResizableDiv.vue) component to this end, to make it a little more flexible -- but tweaking it and wrapping that around the components above is probably the way to go.

**Is your feature request related to a problem? Please describe.**
n/a

**Additional context**
\<snip\>, but without the weird separator:
<img src="https://i.stack.imgur.com/dvFbB.gif"/>
Unity drawers that function like above next to each other:
<img src="http://www.theappguruz.com/app/uploads/2015/06/screenshot-of-inspector.png"/>

**Comment by Geoxor** [comment]
yeah this is a good idea, the queue used to be a sidebar that was resizable with the ResizableDiv.vue component but it was only able to resize right-wards, will attempt this soon
